tools/site-creator/site-creator.js

```diff
@@ -26,6 +26,25 @@ class SiteCreator extends LitElement {
     return `${String(crawlTime / 1000).substring(0, 4)}s`;
   }
 
+  formChanged(e) {
+    const input = e.target;
+    const value = input.value.toLowerCase();
+    input.value = value; // Update the input field with lowercase value
+
+    try {
+      const url = new URL(value);
+      const org = url.pathname.split('/')[1];
+      const repo = url.pathname.split('/')[2];
+
+      if (org && repo && url.hostname === 'github.com') {
+        this._status = null;
+        this._data = { org, repo };
+      }
+    } catch (err) {
+      console.warn('Invalid url', err);
```

Comment:
This will trigger a warning on every key stroke... Not sure we really need that

Reply:
OK